### PR TITLE
use excellent es6

### PR DESCRIPTION
**Pull request by shackbarth**

### CODE REVIEW

File: views/dashboard.js [hunk 1]
```diff
@@ -27,27 +27,27 @@ var Dashboard = React.createClass({
 
     // make sure that we're logged in by trying to query anything
     $.ajax({
-      url: '/' + org + '/browser-api/v1/resources/honorific?count=true',
+      url: `/${org}/browser-api/v1/resources/honorific?count=true`,
```
Comment (tjwebb) [inline views/dashboard.js]: Well, I would still just use `url.resolve()` here, for safety.

Comment (shackbarth) [inline views/dashboard.js]: You mean this: https://www.npmjs.com/package/resolve-url ? This is clientside...

Comment (tjwebb) [inline views/dashboard.js]: http://nodejs.org/docs/latest/api/url.html#url_url_resolve_from_to

Comment (shackbarth) [inline views/dashboard.js]: Yeah good point, I can browserify that.

**Comment by shackbarth** [comment]
resolves #4

File: views/dashboard.js (hunk 1 above)
Comment (shackbarth) [inline views/dashboard.js]: On second thought I'm not sure what that would get us. The org is part of the pathname, not the hostname. I could
```
url.resolve(`/${org}/`, "browser-api/v1/resources/honorific?count=true")
```

But, normally I think about `url.resolve()` as being useful to marry hostnames and pathnames. It's not like you can throw as many arguments as you want in there like you can in `path.resolve()`.

Comment (tjwebb) [inline views/dashboard.js]: url.resolve will parse only a pathname if that's all its given. It doesn't matter that much, I'd mostly do it to ensure everything is urlencoded as a general practice.

**Pull request merged**